### PR TITLE
Add linkedin provider with `social-media/publishing-profile` map

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds LinkedIn provider with `social-media/publishing-profile` map.

## Motivation and Context

We would like to move all local profiles and maps from `superfaceai/social-media-demo` project to Superface registry.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: superface/super.json [hunk 1]
```diff
@@ -851,6 +854,12 @@
       "parameters": {
         "accessToken": "$TWITTER_ACCESS_TOKEN"
       }
+    },
+    "linkedin": {
+      "file": "..//providers/linkedin.json",
```
Comment (martinalbert) [inline superface/super.json:859]: Typo in path

**Review by martinalbert — APPROVED** [review]
Looks good 👍🏻

File: superface/super.json (hunk 1 above)
Comment (jnv) [reply superface/super.json:859]:
```suggestion
      "file": "../providers/linkedin.json",
```

**Review by jnv — APPROVED** [review]
Lovely map ;)

**Pull request merged**